components/DropdownMenu.vue

```diff
@@ -47,14 +49,14 @@ function menuItemClick(event: Event, item: DropdownMenuItem) {
         <p
           v-else-if="item.clickFunction"
           class="flex justify-start items-center gap-1 py-2 px-3"
-          @click="(event) => menuItemClick(event, item)"
+          @click="(event: MouseEvent) => menuItemClick(event, item)"
```

Comment:
Types don't have to be defined after updating `typescript `to v5, but then we run into other type-problems in `components/PageHeading.vue` 😰